sdk/core/azure_core_amqp/README.md

```diff
@@ -8,7 +8,7 @@ This crate is part of a collection of crates: for more information please refer
 
 ## Example
 
-```rust,no_run
+```rust no_run
```

Comment:
Going forward, either will work. While <https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html> makes no mention of separators, I've seen both ways. Rust itself seems to use commas as separators, though spaces work and I've seen elsewhere as well.

Reply:
Oh oops - I thought the new standard was no comma. I'll fix this.

Reply:
Either works, apparently.